src/app/berkeley_migration/berkeley_migration.ml

```diff
@@ -410,25 +410,28 @@ let main ~mainnet_archive_uri ~migrated_archive_uri ~runtime_config_file
                       Archive_lib.Processor.Block.table_name ) )
                 () )
         in
-        let garbage_block_ids_sql =
-          String.concat ~sep:"," @@ List.map garbage_block_ids ~f:Int.to_string
-        in
-        let%bind () =
+        if List.is_empty garbage_block_ids then Deferred.unit
+        else
+          let garbage_block_ids_sql =
+            String.concat ~sep:","
+            @@ List.map garbage_block_ids ~f:Int.to_string
+          in
+          let%bind () =
+            query_migrated_db ~f:(fun (module Conn : CONNECTION) ->
+                Conn.exec
+                  (Caqti_request.exec Caqti_type.unit
+                     (sprintf "DELETE FROM %s WHERE block_id IN (%s)"
```

Comment:
I am a little confused. You are having parenthesis here and removing parenthesis below. Am I missing something?

Reply:
It's a mistake. Fixing.